src/components/accordion/accordionWrapper.js

```diff
@@ -1,12 +1,16 @@
 import styled from 'styled-components'
+import chevronDown from '../../theme/assets/images/icon-chevron-down.svg'
+import chevronUp from '../../theme/assets/images/icon-chevron-up.svg'
 
 const AccordionWrapper = styled.div`
   .accordion__body {
     padding-top: 0 !important;
-    background-color: ${props => props.theme.colors.lightGrey} !important;
+    background: ${props => props.theme.colors.lightGrey};
     animation: none !important;
 
     p {
+      font-size: 18px;
+      line-height: 27px;
       @media (max-width: ${props => props.theme.breakpoints[1]}) {
```

Comment:
avoid max-width BPs, as technically this should be the breakpoint -1px. This will conflict with any min-width styles of the same BP, since they will overlap at that px size.